doc/faq/performance.rst

```diff
@@ -36,7 +36,41 @@ You can configure how often the timestamp should be read using the pi.cfg
 variable ``PI_CHECK_RELOAD_CONFIG``. You can set this to seconds. If you use this
 config value to set values higher than 0, you will improve your performance.
 But: other processes or instances will learn later about configuration changes
-which might lead to unexpected behaviour.
+which might lead to unexpected behavior.
+
+.. _faq_perf_crypto:
+
+Cryptography
+~~~~~~~~~~~~
+
+Cryptography, especially Public-key cryptography is typically based on solving
+difficult and/or time-consuming problems. privacyIDEA uses a lot of cryptographic
+techniques to ensure the security of its operation.
+
+Some cryptographic operations are not strictly necessary for the secure operation
+but provide additional safety for the user. If performance is an issue, some of
+these can be disabled to improve the throughput.
+
+Please also read :ref:`crypto_considerations` to understand the implications.
+
+.. _faq_perf_crypto_audit:
+
+The Audit-log
+^^^^^^^^^^^^^
+
+Each entry in the :ref:`audit` log is signed to detect tampering. When the
+safety of the private key in ``PI_AUDIT_KEY_PRIVATE`` can be assured, the config
```

Comment:
Again. Maybe "if you can be sure, that the private key originates from a trusted source".

Reply:
Done in 91bcff523